Helper/Cacher.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+namespace Reach\Payment\Helper;
+
+//use Magento\Framework\App\Helper\AbstractHelper;
```

Comment:
general question, does this cache exist on the server? or per session? just to confirm my understanding